README.md

```diff
@@ -2,7 +2,7 @@
 
 ![github actions badge](https://github.com/cachix/nix-action/workflows/nix-action%20test/badge.svg)
 
-Installs [Nix](https://nixos.org/nix/) in platform agnostic way (supports: Linux/macOS).
+Installs [Nix](https://nixos.org/nix/) on GitHub Actions for the supported platforms: Linux and macOS.
```

Comment:
Not really agnostic if it knows about the platforms. Also mention GitHub Actions for completeness.